megamek/src/megamek/common/LandAirMech.java

```diff
@@ -2108,4 +2115,82 @@ public void updateSensorOptions() {
             }
         }
     }
+    
+    /**
+     * @return Returns the autoEject.
+     */
+    public boolean isAutoEject() {
+        if (game.getOptions().booleanOption(OptionsConstants.ADVANCED_STRATOPS_QUIRKS)) {
+            return !hasQuirk(OptionsConstants.QUIRK_NEG_NO_EJECT);
+        }
+        return true;
+    }
+
+    /**
+     * @param autoEject
+     *            The autoEject to set.
+     */
+    public void setAutoEject(boolean autoEject) {
+        this.autoEject = autoEject;
+    }
+
+    /**
+     * @return Conditional Ejection Ammo
+     */
+    public boolean isCondEjectAmmo() {
+        return condEjectAmmo;
+    }
+
+    /**
+     * @param condEjectAmmo
+     *            The condEjectAmmo to set.
+     */
+    public void setCondEjectAmmo(boolean condEjectAmmo) {
+        this.condEjectAmmo = condEjectAmmo;
+    }
+
+    /**
+     * @return Conditional Ejection Fuel Explosion
+     */
+    public boolean isCondEjectFuel() {
+        return condEjectFuel;
+    }
+
+    /**
+     * @param condEjectFuel
+     *            The condEjectFuel to set.
+     */
+    public void setCondEjectFuel(boolean condEjectFuel) {
+        this.condEjectFuel = condEjectFuel;
+    }
+
+    /**
+     * @return Conditional Ejection SIDest (Fighter only)
+     */
+    public boolean isCondEjectSIDest() {
+        return false;
```

Comment:
This will cause the LAM pilot not to eject in fighter mode for situations that cause SI damage that destroys the CT.
```suggestion
        return isCondEjectCTDest();
```

Reply:
Fixed.